component/media/css/bootstrap.css

```diff
@@ -90,7 +90,7 @@
 	bottom: -0.25em;
 }
 .jevbootstrap img {
-	width: auto\9;
```

Comment:
See http://stackoverflow.com/questions/8004765/css-9-in-width-property

Were you removing is because it looked strange?  Before removing it please test in MSIE.